packages/cowsay.rb

```diff
@@ -0,0 +1,13 @@
+require 'package'
+
+class Cowsay < Package
+  description 'cowsay is a program that generates ASCII pictures of a cow with a message.'
+  homepage 'https://en.wikipedia.org/wiki/Cowsay'
+  version '3.04'
+  source_url 'https://github.com/tnalpgge/rank-amateur-cowsay/archive/cowsay-3.04.tar.gz'
+  source_sha256 'd8b871332cfc1f0b6c16832ecca413ca0ac14d58626491a6733829e3d655878b'
+
+  def self.install
+    system "sh install.sh #{CREW_PREFIX}"
```

Comment:
Does `crew files cowsay` return any results?  I'm thinking not until you use `#{CREW_DEST_PREFIX}` instead.  BTW, `crew files ...` is a great way to test if the package installed correctly or not.